ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type_l3_interfaces.schema.yml

```
@@ -95,6 +111,7 @@ $defs:
           description: |-
             The WAN Carrier this interface is connected to.
             This is used to infer the path-groups in which this interface should be configured.
+            Unless the Carrier is marked as 'trusted' under `wan_carriers`, `ipv4_acl_in` is also required on all WAN interfaces.
```

Let's change`WAN Carrier` to `WAN carrier` .. I am upper-casing too much sometimes :)
```suggestion
            Unless the carrier is marked as 'trusted' under `wan_carriers`, `ipv4_acl_in` is also required on all WAN interfaces.
```